src/Point.ts

```diff
@@ -1 +1,23 @@
-export class Point {}
+export class Point {
+    public x: number;
+    public y: number;
+
+    constructor(x: number, y: number) {
+        this.x = x || 0;
+        this.y = y || 0;
+    }
```

Comment:
In most of the cases you can declare properties as a constructor argumenst
```suggestion
    constructor(public x: number = 0, public y: number = 0) {}
```